src/status_im/react_native/resources.cljs

```diff
@@ -62,9 +62,7 @@
    :empty-recent        (js/require "./resources/images/ui/empty-recent.png")
    :analytics-image     (js/require "./resources/images/ui/analytics-image.png")
    :welcome-image       (js/require "./resources/images/ui/welcome-image.png")
-   :lock                {:image  (js/require "./resources/images/ui/lock.png")
-                         :width  993
-                         :height 933}
+   :lock                (js/require "./resources/images/ui/lock.png")
```

Comment:
out of curiosity, why is the lock a png and not a vector file?

Reply:
it's a complex image, I suppose it is the same as the tribute to talk one, in svg it wouldn't render properly?